### ISSUE TITLE
Ability to add short comment / readme to environment

### ISSUE THREAD

**Issue by janwinkler1** [conda/conda#11480]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other feature requests and couldn't find a duplicate (including also the ``type-feature`` tag)
- [X] I confirmed that it's not related to another project (see the above section)

### What is the idea?

This feature would allow Conda users to add a brief comment or readme to an environment while creating the environment.

### Why is this needed

The benefit of this feature would be an improved overview of existing environments when the name of the environment is not inclusive enough. 

### What should happen?

A user would type `conda create --name envname -m "comment with max 140 chars"`. I chose the `-m` flag to highlight the similarity to the usage of a `git commit` message. 

When the user created the environment, upon typing  `conda env list` the comment would show up in that list. 

`envname           /usr/path/to/envname         "meaningful comment"`
`envname1         /usr/path/to/envname1         "another meaningful comment"`

A lighter version of that could be that the message/readme only appears upon calling `conda info` when the environment is activated. 

Additionally, the comment could be passed on when sharing an environment. 

### Additional Context

I regularly work with conda environments, and I adore it. It makes things so much easier. When I develop ideas for a new project or thesis, trying to find the best way on what packages to use, I sometimes create multiple environments to see what works best. Especially with the current M1 situation, where sometimes you have to install packages in a rosetta environment from different sources, it would be helpful to add the mentioned feature. Moreover, looking into the future, I assume this could be helpful for users working on various platforms, too, as I presume that collaboration in projects and thus sharing the environments will continue to expand.

**Comment by beeankha** [comment]
Hello @janwinkler1 , thank you for filing this issue and also for the kind words. It's really great that you're enjoying using `conda`, and we appreciate you letting us know about that!

There are a couple of ways we can approach the possible implementation of this new feature; if you feel up for it, you are welcome to write this code and submit a pull request. Another possibility is that this feature could work as a plugin. There is currently a [work in progress PR](https://github.com/conda/conda/pull/11435) that, once merged, will enable plugin integration into `conda` and a new subcommand that adds comments to `conda` envs would be a great way to take advantage of that infrastructure!

Whichever route this ends up going, I've applied the `plugins` label to this and have noted your suggestion as a great use-case for plugins. Thanks again for engaging with us and submitting this issue!

**Comment by SaschaFroelich** [comment]
Has there been any progress on this front? Would love to be able to add comments/ tags/ readmes to envs.

**Comment by janwinkler1** [comment]
hi sascha, not from my side, i did not find the time.. maybe someone else has done something in the meantime?

**Comment by philipmathieuKM** [comment]
I agree that this would be a great feature. I am also tagging my personal account @philipmathieu in case I have the time to follow up and contribute
